setup.sh

```diff
@@ -37,6 +37,20 @@ else
 	echo "You are not logged into the Heroku CLI. OAUTH keys have not been set, so you will be unable to log in with your email."
 fi
 
+# Pull AWS S3 credentials
+if ! command -v aws 1>/dev/null
+then
+	echo "Did not find AWS CLI installation. You will be unable to view/upload profile pictures for users."
+elif ! aws sts get-caller-identity | grep "csm-web-s3-user" 1>/dev/null
+then
+	echo "You are not logged into the AWS CLI with the credentials for 'csm-web-s3-user'. You will be unable to view/upload profile pictures for users."
+	echo "If calling 'aws sts get-caller-identity' prints 'Partial credentials found in env', check your environment variables. It may be overriding your credentials in ~/.aws."
+else
+	echo "AWS_STORAGE_BUCKET_NAME=csm-web-media" >> .env
+	echo 'AWS_ACCESS_KEY_ID=$(aws configure get aws_access_key_id)' >> .env
```

Comment:
This is single-quoted, so the `$(...)` will not actually be run, but included verbatim in .env